devices/usbc-cdc/usbphy-imx6ull.c

```diff
@@ -50,19 +66,44 @@ void *phy_getBase(void)
 
 u32 phy_getIrq(void)
 {
-	return 75;
+	return 75u;
 }
 
 
+static void phy_setClock(void)
+{
+	imx6ull_setDevClock(clk_usboh3, 0x03);
+
+	*CCM_ANALOG_PLL_USB0 |= CCM_ANALOG_PLL_USB0_ENABLE;
+	*CCM_ANALOG_PLL_USB0 |= CCM_ANALOG_PLL_USB0_POWER;
+	while ((*CCM_ANALOG_PLL_USB0 & CCM_ANALOG_PLL_USB0_LOCK) == 0u) {
+	}
+	*CCM_ANALOG_PLL_USB0 &= ~(CCM_ANALOG_PLL_USB0_BYPASS);
+	*CCM_ANALOG_PLL_USB0 |= CCM_ANALOG_PLL_USB0_EN_USB_CLKS;
+}
+
 void phy_reset(void)
 {
-	/* TODO: reset phy controller */
+	USB0_PHY[USB_PHY_CTRL] |= USB_PHY_CTRL_SFTRST;
```

Comment:
Our convention:
base address is kept in common struct, access to registers is performed via + operator:
```suggestion
	*(phyusb_common.base + USB_PHY_CTRL) |= USB_PHY_CTRL_SFTRST;
```

Reply:
`phyusb_common.base` is to be initialized in `phy_init()`

Reply:
Example: https://github.com/phoenix-rtos/plo/blob/f51d9744165a6914e631c83543282fa1c7b80348/devices/usbc-cdc/usbphy-imxrt117x.c#L101

Reply:
Is it good now?

Reply:
`CCM_ANALOG_PLL_USB0` stayed in a different style